src/components/Banner.tsx

```diff
@@ -0,0 +1,63 @@
+import { useState, useEffect } from 'react';
+import { ShowData } from '../types';
+import { Typography as Typ } from '@mui/material';
+import SearchInput from './SearchInput';
+
+interface BannerProps {
+    /**
+     * Array of data to randomly choose an image
+     */
+    data: ShowData[] | null;
+    /**
+     * Title that is displayed on the banner
+     */
+    title: string;
+    /**
+     * If true, renders SearchInput component below title
+     */
+    renderSearchInput?: boolean;
+    /**
+     * If true, renders Streamability logo
+     */
+    renderLogo?: boolean;
+}
+
+/**
+ * Image Banner with title and optional search input
+ * @returns {JSX.Element} | Banner
+ */
+/* eslint-disable react/prop-types */
```

Comment:
@Thenlie Checkout this ignore.  https://stackoverflow.com/questions/38684925/react-eslint-error-missing-in-props-validation

Unfamiliar with this error/package, let me know your thoughts.

Reply:
I'm not sure why this is coming up but we don't need it so you can add `"react/prop-types": "off"` to the eslint rules and remove the ignore comment.